lib/ansible/modules/web_infrastructure/letsencrypt.py

```diff
@@ -950,7 +964,10 @@ def main():
     module.run_command_environ_update = dict(LANG='C', LC_ALL='C', LC_MESSAGES='C', LC_CTYPE='C')
     locale.setlocale(locale.LC_ALL, 'C')
 
-    cert_days = get_cert_days(module, module.params['dest'])
+    if module.params.get('dest'):
```

Comment:
so it looks like they fullchain_dest and dest are also mutually exclusive, aren't they? require one of but not both?

Reply:
Both `dest` and `fullchain_dest` are not required, but at least one of them must be specified. Both can be specified as well. If both are specified, the module assumes that both are equivalent for determining whether the certificate is new enough (if `dest` is given, the certificate it points to is checked, otherwise the certificate where `fullchain_dest` points to).

While only one of them is required, both will be created (if both are specified) if a new certificate is retrieved. So the user can decide whether she wants to work with leaf certificates, fullchain certificates, or both (without the need to manually concatenating or splitting files).

Reply:
I see.